src/include/duckdb/parser/parsed_data/sequence_value_info.hpp

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/parser/parsed_data/sequence_value_info.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/parser/parsed_data/parse_info.hpp"
+
+namespace duckdb {
+
+struct SequenceValueInfo : public ParseInfo {
+public:
+	static constexpr const ParseInfoType TYPE = ParseInfoType::SEQUENCE_VALUE_INFO;
+
+public:
+	SequenceValueInfo() : ParseInfo(TYPE) {};
+
+	//! Schema name
+	string schema;
+	//! Sequence name
+	string name;
+	//! Usage count
+	uint64_t usage_count;
+	//! Counter value
+	int64_t counter;
+
+public:
+	unique_ptr<SequenceValueInfo> Copy();
+	string ToString() const;
```

Comment:
This doesn't look like it's defined anywhere?

Reply:
Resolved in the first PR on the stack